app/uk/gov/hmrc/plasticpackagingtax/registration/forms/ProcessMoreWeight.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms
+
+import play.api.data.Form
+import play.api.data.Forms.{mapping, text}
+
+case class ProcessMoreWeight(answer: Option[Boolean])
+
+object ProcessMoreWeight extends CommonFormValidators {
```

Comment:
I wonder whether we could use a better name here? Does `expectToExceedThresholdIn30d` work better?

Reply:
Does expectToExceedThresholdWeight be better do we need to mention 30d in the class or variable name?

Reply:
How about `expectToExceedThresholdSoon`?